### PR TITLE
move constants to website directory

### PR DESCRIPTION
**Pull request by swimj**
I'm not sure what the best way to handle the BATCH_SIZE and MAX_TRAIN_SIZE parameters. They're class level variables, used in static methods, so changing these into a constants passed in from the website directory would actually require some nontrivial design changes.

### CODE REVIEW

File: server/website/website/tasks/async_tasks.py [hunk 1]
```diff
@@ -450,7 +459,7 @@ def map_workload(target_data):
             # the knob configurations attempted so far by the target.
             y_col = y_col.reshape(-1, 1)
             model = GPRNP()
```
Comment (bohanjason) [inline server/website/website/tasks/async_tasks.py]: Could you please pass the parameters to GPRNP here ? e.g. length_scale=DEFAULT_LENGTH_SCALE, magnitude=DEFAULT_MAGNITUDE..   MAX_TRAIN_SIZE , BATCH_SIZE

File: server/analysis/gp.py [hunk 2]
```diff
@@ -20,8 +20,7 @@ class GPRNP(object):
     MAX_TRAIN_SIZE = 7000
     BATCH_SIZE = 3000
```
Comment (bohanjason) [inline server/analysis/gp.py]: Could you please remove these parameters and pass them at the asynchronous task. You may also want to add the two parameters in the _init_() function.

Comment (swimj) [reply server/analysis/gp.py]: MAX_TRAIN_SIZE is currently used as a class data member in check_X_y, which is a static method.  So if we only pass in MAX_TRAIN_SIZE through __init__, and save it as an instance variable like all the others, then I'm not sure how check_X_y can know MAX_TRAIN_SIZE.  Advice?

Comment (bohanjason) [reply server/analysis/gp.py]: Can we change the static method  check_X_y to a normal function (i.e. remove the static method decorator) ? What do you think ?

Comment (swimj) [reply server/analysis/gp.py]: Yes, that would work.  I'm not familiar enough with Python internals to know whether removing staticmethod has any performance impacts or is purely for visual convenience, but if you're good with it then I am. Will fix after today's meeting.

**Pull request merged**